pallets/protos/src/benchmarking.rs

```diff
@@ -20,6 +20,11 @@ fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
 }
 
 benchmarks! {
+
+	where_clause { where
```

Comment:
what does this do?

Reply:
It allows us to further bound `T` (https://docs.rs/frame-benchmarking/2.0.0/frame_benchmarking/macro.benchmarks.html)

Since I bounded `T::AccountId` to `AsRef<[u8]>` in the implementation block of `Pallet<T>` (in `pallets/protos/lib.rs`), that's why we also need to bound it when benchmarking